install/website-chart.js.2.5.0.patch

```diff
@@ -0,0 +1,51 @@
+--- Chart.js	2017-05-25 09:56:43.000000000 +0200
++++ website/js/Chart.js	2017-10-18 09:22:45.000000000 +0200
+@@ -8868,6 +8868,7 @@
+ 			beforeLabel: helpers.noop,
+ 			label: function(tooltipItem, data) {
+ 				var datasetLabel = data.datasets[tooltipItem.datasetIndex].label || '';
++				var customLabel = data.datasets[tooltipItem.datasetIndex].data[tooltipItem.index].label || ''; 
+ 				return datasetLabel + ': ' + tooltipItem.yLabel;
+ 			},
+ 			labelColor: function(tooltipItem, chartInstance) {
+@@ -8886,7 +8887,11 @@
+ 
+ 			// Args are: (tooltipItems, data)
+ 			beforeFooter: helpers.noop,
+-			footer: helpers.noop,
++			footer: function(tooltipItem, data) {
```

Comment:
same as above... this in particular looks very amenable to just installing the function yourself – that's probably the library's intention